pkgs/by-name/tr/transmission_4-macos-bin/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  _7zz,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "transmission";
```

would it be helpful if this package was named differently to avoid possible name collisions?
```suggestion
  pname = "transmission_4-bin";
```